person.rb

```diff
@@ -1,21 +1,25 @@
 require './nameable'
 
 class Person < Nameable
+  attr_accessor :name, :age
+  attr_reader :id, :rentals
+
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..100)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rental = []
```

Comment:
- [ ] @rental needs to be @rentals, kindly update this typo

Reply:
@GzimAsani,
thanks for your good and valuable feedback,
I have considered your comment into account.